internal/services/loadbalancer/rule_resource.go

```diff
@@ -458,3 +357,123 @@ func expandAzureRmLoadBalancerRule(d *pluginsdk.ResourceData, lb *network.LoadBa
 		LoadBalancingRulePropertiesFormat: &properties,
 	}, nil
 }
+
+func resourceArmLoadBalancerSchema() map[string]*pluginsdk.Schema {
+	out := map[string]*pluginsdk.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: loadBalancerValidate.RuleName,
+		},
+
+		"resource_group_name": azure.SchemaResourceGroupName(),
```

Comment:
I believe there was a `// TODO` comment missing here for deprecating and removing `resource_group_name` (like for the data source) since it can be inferred from the `loadbalancer_id`

Reply:
Yup, looks like I missed a few TODO items for this package. I'll push them as well